src/alpaca_typer.erl

```diff
@@ -4760,19 +4781,7 @@ different_arity_test_() ->
               ?assertMatch(
                  {error, {not_found, _, <<"add">>, 2}},
                  module_typ_and_parse(Code))
-      end
-    , fun() ->
```

Comment:
Ah, of course, I should have thought through this a bit more.  Thanks for the explanation.